lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -231,6 +232,19 @@ def process_action(*args)
       ensure
         Datadog::Contrib::Rails::ActionController.finish_processing(payload)
       end
+
+      # rubocop:disable Style/EmptyElse
+      def response_status
+        case response
+        when ActionDispatch::Response
+          response.status
+        when Array
+          status = response.first
```

Comment:
the first element always contain the status? If yes, can you add a comment explaining why the first and if we have to change this behavior in the future?

Reply:
Yes, it's always the first because that's the Rack response standard: status code, headers, body.